src/components/button/index.tsx

```diff
@@ -0,0 +1,20 @@
+interface ButtonProps {
+  children: string;
+  color: string;
+  onClick?: () => void;
+  disabled?: boolean;
+}
+
+const Button = ({ children, color, onClick, disabled }: ButtonProps) => {
```

Comment:
O que você acha de por uma cor padrão para esse botão caso quem for usar esse botão não mandar a cor?
```suggestion
const Button = ({ children, color = 'bg-red-600', onClick, disabled }: ButtonProps) => {
```